docs/guides/platform/managed/linode-managed/index.md

```diff
@@ -231,22 +254,30 @@ A Managed service can only be linked to a contact group and not to individual co
 
 1.  Fill out the form and click the **Save** button.
 
-## Services
+## Configuring Monitors
 
 Linode Managed monitors the *services* running on your Linodes. Setting up services is an essential step in the configuration process--Linode Managed can't monitor anything until you add services to be monitored.
 
-### Adding a New Service
+### Adding a New Monitor
 
 {{< content "managed-add-service-shortguide" >}}
 
-### Temporarily Disabling Service Monitoring
+### Temporarily Disabling Monitors
 
 {{< content "managed-disable-service-shortguide" >}}
 
-### Reenabling Disabled Services
+### Reenabling Disabled Monitors
 
 {{< content "managed-reenable-service-shortguide" >}}
 
-### Removing Services
+### Removing Monitors
 
 {{< content "managed-remove-service-shortguide" >}}
+
+## Optional: Installing cPanel
+
+Each Linode Compute Instance on a Managed account is eligible to receive a [cPanel](https://cpanel.net/) license at no additional charge.
+
+1. **Installing cPanel:** The easiest method to quickly get cPanel up and running is to deploy a new Linode using the [cPanel Marketplace App](/docs/guides/how-to-deploy-cpanel-with-marketplace-apps/). If this Marketplace App does not support your desired Linux distribution, you can also follow the instructions within the [Install cPanel on CentOS](/docs/guides/install-cpanel-on-centos/) guide or the [Installation Guide](https://docs.cpanel.net/installation-guide/) within cPanel's documentation. It's recommended to only install cPanel on the following Linux distributions: CentOS 7, AlmaLinux 8, Rocky Linux 8, or Cloud Linux. CentOS 8 is not recommended due to the limited support timeline.
```

Comment:
@wildmanonline I was thinking we should maybe link to https://www.linode.com/marketplace/apps/cpanel/cpanel/ instead of, or in addition to, the cpanel marketplace guide, and I want to hear your opinions on this. My reasoning:

- It has the "deploy this app" button to direct users to the appropriate page in the cloud manager
- The cPanel marketplace guide doesn't really show the reader where the marketplace options are in the cloud manager. To me, it reads as a companion piece to someone who is already in the cPanel Marketplace app flow in the cloud manager (and clicks through to the guide from the manager). It doesn't seem as helpful to someone who has never heard of marketplace apps before, which may be the case for someone reading about Linode Managed in this guide.

On that second point, it might be a good idea to update the cPanel marketplace guide (or all the marketplace guides?) with a link to https://www.linode.com/docs/guides/how-to-use-marketplace-apps-at-linode/ and a quick general note about what Marketplace apps are for at the top. But, that would be a separate issue.

Reply:
I like this suggestion. Let me do a quick commit with this change...